src/vs/workbench/services/suggest/browser/simpleCompletionModel.ts

```diff
@@ -116,7 +116,7 @@ export class SimpleCompletionModel<T extends SimpleCompletionItem> {
 			const overwriteBefore = item.completion.replacementLength; // item.position.column - item.editStart.column;
 			const wordLen = overwriteBefore + characterCountDelta; // - (item.position.column - this._column);
 			if (word.length !== wordLen) {
-				word = wordLen === 0 ? '' : leadingLineContent.slice(-wordLen);
+				word = wordLen === 0 ? '' : leadingLineContent.slice(-wordLen).replaceAll('\n', '');
```

Comment:
This doesn't look right to me, `leadingLineContent.slice(-wordLen)` should never contain a `\n` as it's a word separator?